onecore_maintenance_extension/__manifest__.py

```diff
@@ -21,7 +21,8 @@
     ],
     'assets': {
         'web.assets_backend': [
-            'onecore_maintenance_extension/static/src/views/*.xml',
+            'onecore_maintenance_extension/static/src/views/chatter.xml',
+            'onecore_maintenance_extension/static/src/views/maintenance_request_kanban.xml',
```

Comment:
Perhaps rename this file to maintenance_request_item or card or something since it's used in both kanban view and mobile view?